packages/runner/src/suite.ts

```diff
@@ -215,6 +217,8 @@ function createSuite() {
           ? suite(formatTitle(name, items, idx), () => fn(...items), options)
           : suite(formatTitle(name, items, idx), () => fn(i), options)
       })
+
+      this.withContext({ each: undefined })
```

Comment:
I don't like this usage of the function. Maybe just `delete this.each` or `this.each = undefined`?

Reply:
we can't do that as `this` reference to a bound function(`suite` with context), so here `this.each` is the function(`suite.each`), not the flag `each`.